CMakeLists.txt

```diff
@@ -319,6 +325,22 @@ if (MRDOCS_BUILD_TESTS)
     endif()
 endif()
 
+#-------------------------------------------------
+#
+# Copy libc++ and clang headers
+#
+#-------------------------------------------------
+
+set(MRDOCS_SHARE_LIBCXX_DIR "${CMAKE_CURRENT_BINARY_DIR}/../share/mrdocs/libcxx")
+message(STATUS "Copying libc++ headers to ${MRDOCS_SHARE_LIBCXX_DIR}")
+file(MAKE_DIRECTORY ${MRDOCS_SHARE_LIBCXX_DIR})
+file(COPY ${LIBCXX_DIR}/ DESTINATION ${MRDOCS_SHARE_LIBCXX_DIR})
```

Comment:
The files are placed here because they're needed not only for `if (MRDOCS_INSTALL)` but also during the `if (MRDOCS_BUILD_DOCS)` step for documentation generation. We need them accessible in both contexts.

Reply:
OK. I'm assuming the MRDOCS_BUILD_DOCS uses the addons option. So let's just make the libcxx config option public then. This way we can avoid copying the files, which is bad workaround. 

The `if (MRDOCS_INSTALL)` step also needs a step to install these files, IIRC. If you're not installing these files, the demos can't find them and are probably falling back to something else.

Reply:
👍